h2o-core/src/main/java/water/util/OSUtils.java

```diff
@@ -53,6 +53,10 @@ public static boolean isLinux() {
     return getOsName().toLowerCase().startsWith("linux");
   }
 
+  public static boolean isMac() {
```

Comment:
appears to be unused

Reply:
after all, yes, unused: ended up using the `isLinux` one as multinode is supported only on linux.
Want me to remove it or can it become useful somehow?

Reply:
up to you, doesn't bother me much - impossible to test in current infrastructure though